### PR TITLE
TAR-301 Fix ubuntu installation instructions for EE

### PR DESCRIPTION
**Pull request by seemethere**
Signed-off-by: Eli Uriegas <eli.uriegas@docker.com>

### Proposed changes

Was pointing to a non-existent `$DOCKER_EE_VERSION` repository, and still had
installation instructions for s390x and ppc64el which we do not support
with the latest EE release.

### Related issues (optional)

CC @thaJeztah @ahh-docker @L-Hudson @dave-tucker

### CODE REVIEW

File: install/linux/docker-ee/ubuntu.md [hunk 1]
```diff
@@ -173,47 +173,13 @@ from the repository.
...
-
-    ```
-
-    </div>
-    <div id="ppc64el_repo" class="tab-pane fade" markdown="1">
-
     ```bash
     $ sudo add-apt-repository \
-       "deb [arch=ppc64el] $DOCKER_EE_URL/ubuntu \
+       "deb [arch=$(dpkg --print-architecture)] $DOCKER_EE_URL/ubuntu \
```
Comment (thaJeztah) [inline install/linux/docker-ee/ubuntu.md]: Does this print the correct architecture (i.e. `amd64`, not `x86_64`)?

Comment (seemethere) [reply install/linux/docker-ee/ubuntu.md]: It does, that was the original instructions before we changed them. That's the how the instructions go in the get.docker.com script.

**Comment by ahh-docker** [comment]
LGTM. @L-Hudson can you please review and merge? Thanks!

**Comment by L-Hudson** [comment]
Good to go

**Pull request merged**